tests/test_handlers.py

```diff
@@ -919,3 +919,25 @@ def test_symmetric_am1ccc():
     # set(ref_charges) == {-1.8165082, -1.8158009, 0.90795946}
     # set(test_charges) == {-3.815801, -1.8158009, 0.18349183, 0.90795946}
     np.testing.assert_array_equal(test_charges, ref_charges)
+
+
+def test_harmonic_bonds_complete():
+    """On a test molecule containing [oxygen] ~ [halogen] bonds,
+    assert that either: (1) a ValueError is raised or (2) the number of
+    parameterized bonds equals the number of input bonds"""
+
+    mol = Chem.MolFromSmiles("O(F)F")  # 0 smirks matches using current (2022-05-16) handler
+
+    ff = Forcefield.load_from_file(DEFAULT_FF)
+    parameterize = ff.hb_handle.parameterize
+
+    try:
```

Comment:
I find this test confusing. What lines actually run and which raise exceptions?

Reply:
Currently, just one branch runs. (The one where pytest expects a ValueError.)

I initially included a branch in case the handler is modified to another correct behavior in the future (return 2 bonds: pass, raise ValueError: pass, return 0 bonds: fail)

Simplified in https://github.com/proteneer/timemachine/pull/742/commits/7608223abf871d472fb2340b8e98c985877c5835 to (raise ValueError: pass, else: fail)